.github/workflows/ci.yaml

```diff
@@ -406,3 +406,60 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy-scan-image:
+    runs-on: ubuntu-20.04
+    needs: docker-amd64
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download release images
+        uses: actions/download-artifact@v2
+        with:
+          name: release-images
+          path: ./release-images
+
+      - name: Run Trivy vulnerability scanner in image mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          input: "./release-images/code-server-amd64-*.tar"
+          scan-type: "image"
+          ignore-unfixed: true
+          format: "template"
+          template: "@/contrib/sarif.tpl"
+          output: "trivy-image-results.sarif"
+          severity: "CRITICAL"
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: "trivy-image-results.sarif"
+
+  # We have to use two trivy jobs
+  # because GitHub only allows
+  # codeql/upload-sarif action per job
+  trivy-scan-repo:
+    runs-on: ubuntu-20.04
+    needs: docker-amd64
```

Comment:
I don't think we need the Docker image in this step;
also question: does using trivy for our scans kind of obsolete "audit-ci"?

Reply:
No it doesn't, these tools all use different feeds and work in different ways, so they are complementary - I've seen that Dependabot sometimes detects things that Trivy misses, and vice-versa, and sometimes `yarn audit` shows other stuff too. I am not too sure why this happens, but it seems worthwhile to have multiple scans, especially given that the cost of running these additional scans is low

Reply:
Ah, that makes sense - though I'd definitely prefer one tool that does it all rather than many of them that do same but not exactly the same things from a maintainability perspective.

I guess having multiple tools works, at least in the short term!

Reply:
Oh yes! When I split up the two jobs, I forgot to remove the "needs" step. You're right. Since it's just checking the repo code, it doesn't need the docker step. Will update!